ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -83,6 +83,9 @@
             // before initializing the form container load all the locale specific json resources
             // for runtime
             const formLanguage = formContainer.getLang();
+//            if (formContainer._element && !formContainer._element.dir && formLanguage) {
```

Comment:
Please remove this, not sure why is it commented, let's keep the code source clean